contact.html

```diff
@@ -3,148 +3,104 @@
     <head>
         <meta charset="utf-8">
         <title>YouClap - Challenge The World</title>
-        <!-- Stylesheets -->
-        <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
-        <link href="css/style.css" rel="stylesheet">
+        <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" async>
```

Comment:
we can remove the async, this doesn't work for `link` tag, but we can use it in the `script` ones.

Apply this for all the files please 🙏